agent/agent_endpoint.go

```diff
@@ -1359,7 +1360,7 @@ func (s *HTTPServer) AgentConnectCALeafCert(resp http.ResponseWriter, req *http.
 	args.MaxQueryTime = qOpts.MaxQueryTime
 	args.Token = qOpts.Token
 
-	raw, m, err := s.agent.cache.Get(cachetype.ConnectCALeafName, &args)
+	raw, m, err := s.agent.cache.Get(context.Background(), cachetype.ConnectCALeafName, &args)
```

Comment:
Should this be `req.Context()` ?

Reply:
Yep, I though I went back and got all of these but missed one.